index.ios.js

```diff
@@ -172,7 +183,12 @@ var SaveTheShoes = React.createClass({
       barPressure: '110',
       timerRunning: false,
       inTime: null,
-      timeRemaining: Moment.duration(0)
+      timeRemaining: Moment.duration(0),
+      alarmsRemaining: [
+        {alarmSound: 'relief_assembly.mp3', offset: 15},
```

Comment:
I would so like to see 15, etc  as a constant: isn't it being used hither and thither?